tripal_chado/src/Plugin/ChadoBuddy/ChadoCvtermBuddy.php

```diff
@@ -16,15 +17,91 @@
  */
 class ChadoCvtermBuddy extends ChadoBuddyPluginBase {
 
+  /**
+   * Keys are the column aliases, and values are the
+   * table aliases and columns for the dbxref buddy.
+   * @var array
+   *
+   */
+  protected array $db_mapping = [
+    'db_id' => 'db.db_id',
```

Comment:
Why do we need this mapping? Can we not just keep the keys as `db.name` rather then `db_name`?

Additionally, it would be good if the valid columns for each table are pulled from ChadoSchema not hard-coded as this would allow better support for multiple chado versions. If you are worried about speed you could cache the result from chado schema and/or add a method to the ChadoSchema class that grabs the valid columns directly from the YAML to avoid extracting the info from the array returned from getTableDefn

Reply:
Implemented as you suggested, and values cached. Adding a method to the ChadoSchema class can be a future project.